.github/workflows/test-etcd.yml

```diff
@@ -0,0 +1,39 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+        paths:
+            - 'etcd/**'
+    pull_request:
+        paths:
+            - 'etcd/**'
+name: "Tests Etcd"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                go-version:
+                    - 1.19.x
+                    - 1.20.x
+        steps:
+            -   name: Fetch Repository
+                uses: actions/checkout@v3
+            -   name: Install etcd
+                run: |
+                    docker pull bitnami/etcd:latest
+                    
+                    docker run -d --name Etcd-server \
```

Comment:
Is there a github-actions we can use to replace this? Even better if it is something that provides cluster support

Reply:
Don't worry, just check and there's no etcd actions in the Marketplace. So using "docker run" is the best way